Bootstrap migration fixups

dennisreimann opened the pull request:
Continuing with the finetuning of #2490. 

Bootstrap v5 introduced a new XXL breakpoint (1400px width) in addition to the upper limit of 1200px before. Removed that as we currently don't need that.

Contains some form updates too. Will extend this with the feedback from @dstrukt and the test deployment.

dennisreimann commented:
Been through the feedback session with @dstrukt now and will take care of the results from the testers channel next. Afterwards this baby is good to go! 

Mostly this has been minor finetunings, but there are these three areas that visually changed a bit more. So here are before and after screenshots …

## Domain mapping

### Before

![domain-mapping-before](https://user-images.githubusercontent.com/886/119542346-bb94db80-bd8f-11eb-990d-1cdbea32e2ce.png)

### After

![domain-mapping-after](https://user-images.githubusercontent.com/886/119542362-bfc0f900-bd8f-11eb-8beb-6e8df5edb1a9.png)

## Checkout experience

### Before

![checkout-experience-before](https://user-images.githubusercontent.com/886/119542380-c51e4380-bd8f-11eb-8ae4-add656aff333.png)

### After

![checkout-experience-after](https://user-images.githubusercontent.com/886/119542399-ca7b8e00-bd8f-11eb-96b7-892f31bcb5ec.png)

## Webhook

### Before

![webhook-before](https://user-images.githubusercontent.com/886/119542422-cf404200-bd8f-11eb-9173-02f3bc6d98a1.png)

### After

![webhook-after](https://user-images.githubusercontent.com/886/119542432-d2d3c900-bd8f-11eb-9f56-d5c20469a7c4.png)

dstrukt commented:
I want to mess around with the domain mapping a bit.

Should the fee confirmation be split into it's own section "On-chain" .. or does it make sense to keep it under "General". Regardless, I definitely like that the section is broken up to make it easier to digest.

Need to go through this one more time, but everything else is lookin' good.

dennisreimann commented:
@dstrukt yes, let's use this as an example where you can do the pixel pushing and we can have a session where I explain things and help if needed. 👍

dennisreimann commented:
Addressed the issues brought up via the Testers channel (see [Google doc](https://docs.google.com/document/d/1eWjcNRL8qoLjw5I3U-A3jGRsdxstJEs4ciFru8GdDy4/edit)) and rebased with current master. Ready for review 🎉

britttttk commented on BTCPayServer/Views/Stores/ModifyWebhook.cshtml:
```diff
@@ -4,76 +4,75 @@
-                </div>
+                <label asp-for="AutomaticRedelivery" class="form-label">Automatic redelivery</label>
+                <input asp-for="AutomaticRedelivery" type="checkbox" class="btcpay-toggle ms-2" />                
+                <p class="text-muted small form-text">We will try to redeliver any failed delivery after 10 seconds, 1 minutes and up to 6 times after 10 minutes</p>
```
```suggestion
                <p class="text-muted small form-text">We will try to redeliver any failed delivery after 10 seconds, 1 minute and up to 6 times after 10 minutes</p>
```

dennisreimann replied:
Thanks for spotting this! :+1:

dennisreimann commented:
This is good to go. 

@Kukks can you give it a review and merge? @dstrukt could use this for his finetuning PRs :)
